__tests__/app.test.js

```diff
@@ -3,33 +3,59 @@ const app = require("../app.js");
 const seed = require("../db/seeds/seed");
 const testData = require("../db/data/test-data");
 const connection = require("../db/connection.js");
+require("jest-sorted");
 
 beforeEach(() => seed(testData));
 afterAll(() => connection.end());
 
 describe("/api/categories", () => {
   test("GET - 200 responds with an array of category objects", () => {
     return request(app)
-        .get("/api/categories")
-        .expect(200)
-        .then(res => {
-            expect(Array.isArray(res.body)).toBe(true);
-            expect(res.body.length > 0).toBe(true);
-            res.body.forEach(category => {
-                expect(category).toHaveProperty("slug");
-                expect(category).toHaveProperty("description");
-            })
-        })
+      .get("/api/categories")
+      .expect(200)
+      .then((res) => {
+        expect(Array.isArray(res.body)).toBe(true);
+        expect(res.body.length > 0).toBe(true);
+        res.body.forEach((category) => {
+          expect(category).toHaveProperty("slug");
+          expect(category).toHaveProperty("description");
+        });
+      });
+  });
+});
+describe("/api/reviews", () => {
+  test("GET - 200 responds with an array of review objects ordered by date descending", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then((res) => {
+        expect(Array.isArray(res.body)).toBe(true);
+        expect(res.body.length > 0).toBe(true);
```

Comment:
We know the number of items in the test database, so don't be afraid to assert a definite value to make sure that your code is pulling everything from the database.